en/config_vtol/vtol_duo_tailsitter_config.md

```diff
@@ -0,0 +1,94 @@
+# Duo-Tailsitter VTOL Configuration
+
+This is the configuration tuning documentation for a "Duo-Tailsitter VTOL".
+
+A tailsitter is essentially a fixed wing vehicle that takes off in a pitched up configuration (essentially sitting on its tail, hence the name) and then transitions to fixed wing flight by pitch down by 90 degrees to achieve the fixed wing configuration.
+The Duo-Tailsitter variant is so-named because it has two motors (a "quad" tailsitter would have four).
+
+Like other VTOLs, the Duo-Tailsitter has two modes of operation: multicopter (MC) mode and fixed wing (FW) mode.
+In MC mode, lift and roll are dependent on the motors, while the pitch and yaw depend on the ailevons.
+In FW mode, yaw depends on differential thrust from the motors, while roll and pitch depend on the ailevons.
+
+For airframe specific documentation and build instructions see [VTOL Framebuilds](../frames_vtol/README.md).
+
+## Firmware & Basic Settings
+
+1. Run *QGroundControl*
+2. Flash the firmware for your current release or master (PX4 `main` branch build).
+3. In the [Frame setup](../config/airframe.md) section select the appropriate VTOL airframe.
+   At time of writing, this must be the [Generic VTOL Tailsitter](../airframes/airframe_reference.md#vtol-tailsitter).
```

Comment:
Note, the airframe reference just shows the one option and specifies no motors. Can we show an actuator config for the Duo case?

Reply:
Yeah Sure. I will attach a diagram as well as actual motor connection photos. 
I'm also thinking of just shooting a "hardware setup" video. Do you think that might be better?

Reply:
Yes, we need a diagram of the Actuator configuration.

> I'm also thinking of just shooting a "hardware setup" video. Do you think that might be better?

No it would not be better for this particular document.

BUT, depending on your frame, it might be good to have it "as well", alongside an airframe build. It does depend on what you have to do with the frame though. You can see examples of other builds here: http://docs.px4.io/main/en/frames_vtol/vtol_quadplane_falcon_vertigo_hybrid_rtf_dropix.html

Reply:
![FCU](https://github.com/PX4/PX4-user_guide/assets/53693363/dd3532e7-d0ab-41a0-94ee-9fd43ceb9802)
![FCU-2](https://github.com/PX4/PX4-user_guide/assets/53693363/4d227b04-ad3a-437c-a078-fb9ed087ebdb)
@hamishwillee what do you think?

Reply:
I understand the actual Motor leads and battery connections aren't visible but those are pretty straightforward?

Reply:
> what do you think?

- I think this is excellent and will be useful.
- Needs the Actuator Config screen as well.
- Ideally should show the motor connections. What I'd hope here is to map those to the image in the airframe reference so people know which connection has to go to what control. I.e. you can map any output, but you still need to make sure that you're controlling the function that you think you are.
- No, battery connections are not needed. Slightly tempting to see them though because we always say "you need a separate BEC but never do we show the whole battery and powering setup".

Reply:
@sfuhrer There is no motor order diagram in the airframe reference or QGC for this frame: https://docs.px4.io/main/en/airframes/airframe_reference.html#vtol-tailsitter or in QGroundControl. 

Should we at least show the default image - which is a duo with two elevons? 
If so, can you sketch an image or tell me which two FW and MC images in the airframe reference to merge and I can try that.

The actuator screen allows from 0 to 12 motors and of course you can add arbitrary flight controls. My assumption is that these will be somehow mapped to  a frame position in some logical way - i.e. clockwise starting from motor 1. I also assume that the directions of motors might all be changed - but it could be that they must be symmetric or something. Can you outline "the rules" for tailsitters?

Reply:
> Should we at least show the default image - which is a duo with two elevons?

Yes! 
![image](https://github.com/PX4/PX4-user_guide/assets/26798987/7f47d86a-185e-47b4-b8cf-b58a0c98a19b)

Something like that would suffice. Best take the wing of the flying wing and add the two motors. Label the motors 1 
(left) and 2 (right)

>I also assume that the directions of motors might all be changed - but it could be that they must be symmetric or something. Can you outline "the rules" for tailsitters?